From 0d1903919abf4472c9fd7954477036c6a8c52677 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Sat, 17 Aug 2019 10:46:04 +0800
Subject: [PATCH 1/2] Revert "arch:arm64:dts: add reg property to reserve
 memory statically"

This reverts commit 9f172d54b901427ad8241662c77d67f9bcc74ce9.

There is no to change all the reserving memory method from dynamically
into statically. Only the dts node with no-map is need to reserve
memory with static method. So, revert this patch, and then add another
patch to improve lossy_decompress node.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index 456c4fea280f..c1a030dca4c1 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -43,14 +43,16 @@
 		/* device specific region for Lossy Decompression */
 		lossy_decompress: linux,lossy_decompress@54000000 {
 			no-map;
-			reg = <0x00000000 0x6d000000 0x0 0x03000000>;
+			alloc-ranges = <0x0 0x6d000000 0x0 0x03000000>;
+			size = <0x0 0x03000000>;
 		};
 
 		/* global autoconfigured region for contiguous allocations */
 		linux,cma@57000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x70000000 0x0 0x8000000>;
+			alloc-ranges = <0x0 0x70000000 0x0 0x8000000>;
+			size = <0x0 0x8000000>;
 			linux,cma-default;
 		};
 
@@ -58,7 +60,8 @@
 		mmp_reserved: linux,multimedia@70000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x78000000 0x0 0x8000000>;
+			alloc-ranges = <0x0 0x78000000 0x0 0x8000000>;
+			size = <0x0 0x8000000>;
 		};
 	};
 
-- 
2.17.1

